data/presets/building/_outbuilding.json

```diff
@@ -0,0 +1,15 @@
+{
+    "icon": "maki-building",
+    "fields": [
+        "{building}"
+    ],
```

Comment:
This is not necessary, as the fields will be inherited from the parent preset by default.

```suggestion
```